server/integration/ensure_server_prefix_test.go

```diff
@@ -1,42 +0,0 @@
-package integration_test
```

Comment:
This one I wouldn't remove because there's no way we can assert that in a fast and easy way using tracetest.